src/layouts/fr/download.html.jade

```diff
@@ -0,0 +1,77 @@
+doctype html
```

Comment:
I don't know `jade` very well, but what is the idea behind a template engine if we have to duplicate files for i18n?